packages/perseus/src/widgets/interactive-graphs/graphs/point.tsx

```diff
@@ -46,6 +46,16 @@ export function UnlimitedPointGraph(props: PointGraphProps) {
         height,
     ]);
     const [[left, top]] = useTransformVectorsToPixels([minX, maxY]);
+    const itemsRef = React.useRef<[SVGElement | null]>([]);
+
+    React.useEffect(() => {
+        const focusedIndex = props.graphState.focusedPointIndex;
+
+        if (focusedIndex !== null && itemsRef.current[focusedIndex] !== null) {
+            itemsRef.current[focusedIndex]?.focus();
```

Comment:
You would think optional chaining isn't needed here because of the checks buuuuuut typescript _insists_ that its necessary. So 🤷‍♀️